### PR TITLE
Breakdown: Add new BCD entries (from mdn-bcd-collector 1.1.4)

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is a breakdown PR containing all of the new BCD entries generated by an [add-new-bcd.js script](https://github.com/foolip/mdn-bcd-collector/blob/main/add-new-bcd.js), which adds missing BCD entries with some support in at least one browser without flags/prefixes, as determined by results from the mdn-bcd-collector project (version 1.1.4).  Many smaller, refined PRs will be created based upon this one.

### CODE REVIEW

File: api/CookieStoreManager.json [hunk 1]
```diff
@@ -0,0 +1,192 @@
+{
+  "api": {
+    "CookieStoreManager": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": "≤87"
```
Comment (foolip) [inline api/CookieStoreManager.json:10]: How do we end up with data like this? Was only Chrome for Android 87 tested, and it was supported there?

Comment (vinyldarkscratch) [reply api/CookieStoreManager.json:10]: That's correct, we only have results for Chrome Android 87.  I plan to obtain more results, however, as I mentioned during our meeting!

**Comment by foolip** [comment]
Will you be submitting PRs for a bunch of this in the coming week? I only have two weeks of work left in the year in which I can do reviews :)

**Comment by vinyldarkscratch** [comment]
I will be, yes -- I've just submitted a few recently, in fact!

**Pull request closed without merging**